README.md

```diff
@@ -23,7 +23,17 @@ generated documentation is available for Arcs Runtime.
    $ ./tools/setup
    ```
 
-2) That's it! (You can skip the next section.)
+2) The script may have issues with the NVM installation. If so, follow the prompts to manually install NVM and they run the setup script again.
+
+3) For MacOS, You will need to have Xcode installed. The script will fail at this point, and prompt you to install Xcode.
```

Comment:
Confused. Why do we need Xcode?

Reply:
I don't know, but it required me to install it. @shaper, can you provide insight?

Reply:
I just spoke with Walter, and it is used to do some of the compilation of npm modules on Mac.